src/network/networkprotocol_translation_file.cpp

```diff
@@ -0,0 +1,16 @@
+fake_function() {
+    // constexpr const char *accessDeniedStrings[SERVER_ACCESSDENIED_MAX]
+    // We cannot run gettext inside a header file, so do this... >_<
```

Comment:
`gettext.h` provides a `N_` macro that should cover this use case:

```cpp
constexpr const char *accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
	N_("Invalid password"),
	N_("Your client sent something the server didn't expect.  Try reconnecting or updating your client."),
	...
};
```

(Note that you should still keep the gettext calls in `gettext(accessDeniedStrings[denyCode])` in `src/network/clientpackethandler.cpp` for the strings to be actually translated.)

Reply:
Done in 5cbf1a856